.github/workflows/static_analysis.yml

```diff
@@ -8,56 +8,57 @@ env:
 jobs:
   static_analysis:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
```

Comment:
This was never actually used in the static analysis workflow. The GitHub workflow linter in Zed complained about it. The formatting changes to the file are also from Zed FYI.